instancio-core/src/main/java/org/instancio/internal/beanvalidation/JavaxAndJakartaBeanValidationHandlerResolver.java

```diff
@@ -61,38 +45,74 @@
 import java.util.Map;
 import java.util.function.Function;
 
+import static org.instancio.internal.util.ExceptionHandler.runIgnoringTheNoClassDefFoundError;
+
 @SuppressWarnings({"PMD.CouplingBetweenObjects", "PMD.ExcessiveImports"})
-final class JakartaBeanValidationHandlerResolver implements AnnotationHandlerResolver {
+final class JavaxAndJakartaBeanValidationHandlerResolver implements AnnotationHandlerResolver {
 
     private final Map<Class<?>, FieldAnnotationHandler> handlerMap;
 
-    private JakartaBeanValidationHandlerResolver() {
+    private JavaxAndJakartaBeanValidationHandlerResolver() {
         this.handlerMap = initHandlers();
     }
 
-    static JakartaBeanValidationHandlerResolver getInstance() {
+    static JavaxAndJakartaBeanValidationHandlerResolver getInstance() {
         return Holder.INSTANCE;
     }
 
     private static Map<Class<?>, FieldAnnotationHandler> initHandlers() {
         final Map<Class<?>, FieldAnnotationHandler> map = new HashMap<>();
-        map.put(DecimalMax.class, new DecimalMaxHandler());
-        map.put(DecimalMin.class, new DecimalMinHandler());
-        map.put(Digits.class, new DigitsHandler());
-        map.put(Future.class, new FutureHandler());
-        map.put(FutureOrPresent.class, new FutureHandler());
-        map.put(Max.class, new MaxHandler());
-        map.put(Min.class, new MinHandler());
-        map.put(Negative.class, new NegativeHandler(new BigDecimal("-0.5")));
-        map.put(NotBlank.class, new NotEmptyHandler());
-        map.put(NotEmpty.class, new NotEmptyHandler());
-        map.put(NotNull.class, new NotNullHandler());
-        map.put(NegativeOrZero.class, new NegativeHandler(BigDecimal.ZERO));
-        map.put(Past.class, new PastHandler());
-        map.put(PastOrPresent.class, new PastHandler());
-        map.put(Positive.class, new PositiveHandler(new BigDecimal("0.5")));
-        map.put(PositiveOrZero.class, new PositiveHandler(BigDecimal.ZERO));
-        map.put(Size.class, new SizeHandler());
+        runIgnoringTheNoClassDefFoundError(() -> map.put(javax.validation.constraints.DecimalMax.class, new DecimalMaxHandler()));
+        runIgnoringTheNoClassDefFoundError(() -> map.put(jakarta.validation.constraints.DecimalMax.class, new DecimalMaxHandler()));
+
```

Comment:
I think we should avoid catching and ignoring errors. My thinking is this:

- We will either see `javax` or `jakarta` on the classpath, but not both. Therefore,
  - if we see `jarkarta` we handle it using `JakartaBeanValidationHandlerResolver` (we restore the deleted class)
  - if we see `javax` we handle it using `JavaxBeanValidationHandlerResolver`

Reply:
Ok, It seems that it makes sense. I will make required changes